element-hq/element-x-android
# Clear existing notification when a room is marked as read

## Content


First commit are cleanup and small rework.
Last commit is to clear existing notification when a room is marked as read

## Motivation and context


Clear existing notification when a room is marked as read (context: #3184)

## Screenshots / GIFs



## Tests



- Navigate to the room list
- Receive a message. A notification is displayed and the room is unread
- Long click on the room and select "Mark as read"
- Observe that the room is now read (green dot disappear) and notification is dismissed.

## Tested devices

- OS version(s):

## Checklist
